test/inductor/test_mkldnn_pattern_matcher.py

```diff
@@ -149,6 +150,11 @@ def _test_common(
     ):
         counters.clear()
         torch._dynamo.reset()
+        is_xpu = False
+        for input in inputs:
+            is_xpu = is_xpu or (
+                isinstance(input, torch.Tensor) and input.device.type == "xpu"
+            )
```

Comment:
Do we need support the input tensors on different devices?

Reply:
The intention here is not to check different devices. Here is we want to find a hint to choose what quantizer we use. If a XPU tensor exists in inputs, we will use XPUQuantizer.

Reply:
```python
is_xpu = any(
    isinstance(input, torch.Tensor) and input.device.type == "xpu"
    for input in inputs
)
```

Reply:
thanks for suggestions, this logic is moved into https://github.com/pytorch/pytorch/pull/139578/files#diff-a075bab09edbaf789a4ffdebf4eacf70004c21ff58466f6e3502fa14869b8cc0R2946